photonlib-java-examples/src/main/java/org/photonlib/examples/simposeest/robot/Constants.java

```diff
@@ -94,9 +94,14 @@ public class Constants {
     public static final double kFarTgtXPos = Units.feetToMeters(54);
     public static final double kFarTgtYPos =
             Units.feetToMeters(27 / 2) - Units.inchesToMeters(43.75) - Units.inchesToMeters(48.0 / 2.0);
-    public static final Pose2d kFarTargetPose =
-            new Pose2d(new Translation2d(kFarTgtXPos, kFarTgtYPos), new Rotation2d(0.0));
+    public static final double kFarTgtZPos =
+            (Units.inchesToMeters(98.19) - targetHeight) / 2 + targetHeight;
+
+    public static final Pose3d kFarTargetPose =
+            new Pose3d(
+                    new Translation3d(kFarTgtXPos, kFarTgtYPos, kFarTgtZPos), new Rotation3d(0.0, 0.0, 0.0));
 
```

Comment:
I'm pretty sure that the z angle should be 180 degrees, right?

Reply:
Yes, that sounds right. Sorry for the delay, I wanted to run the latest photonvision server to confirm since these rotations have changed since beta-3.